repositories.go

```diff
@@ -138,6 +139,7 @@ func (s *RepositoriesService) RawBlobContent(pid interface{}, sha string, option
 type ArchiveOptions struct {
 	Format *string `url:"-" json:"-"`
 	SHA    *string `url:"sha,omitempty" json:"sha,omitempty"`
+	Writer *io.Writer `url:"-" json:"-"`
```

Comment:
It looks like your code is not formatted using `gofmt`

Reply:
Sorry, my IDE was not configured correctly with this project, it's fixed